### PR TITLE
Add flake8

### PR DESCRIPTION
**Pull request by mayasa3**
Install flake8 and create flake8 file

### CODE REVIEW

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Flake8
+on: pull_request
+job:
+flake8:
```
Comment (bardielle) [inline .github/workflows/flake8.yml]: please use indentations

Comment (mayasa3) [reply .github/workflows/flake8.yml]: It should've had indentations already. I reuploaded the file, is it showing up right?

Comment (bardielle) [reply .github/workflows/flake8.yml]: yes now it's looking good 
Pay attention - this is only a review on your code but I can't validate that it's actually working
Are you sure it's working on your ENV?

Comment (mayasa3) [reply .github/workflows/flake8.yml]: It looks like it's working on my end

**Comment by bardielle** [comment]
Please squash your comments

**Comment by bardielle** [comment]
@mayasa3 Can you merge those two commits into one commit?

**Review by bardielle — APPROVED** [review]

**Pull request merged**